### PR TITLE
Variables for Location and Storage class 

### PR DESCRIPTION
**Pull request by kiran002**
Reopened PR #14

Also addresses the issue #4

### CODE REVIEW

**Comment by kiran002** [comment]
@morgante can you please make the logs available for me? I want to see why it failed, thanks!

**Comment by morgante** [comment]
@kiran002 For now, here's the error:

```
       Error: module.log_export.module.destination.google_bigquery_dataset.dataset: expected location to be one of [US EU asia-east1 asia-northeast1 asia-southeast1 australia-southeast1 europe-north1 europe-west2 us-east4], got US}
```

Which admittedly looks a little odd.

File: modules/bigquery/main.tf [hunk 1]
```diff
@@ -38,7 +38,7 @@ resource "google_project_service" "enable_destination_api" {
 resource "google_bigquery_dataset" "dataset" {
   dataset_id = "${var.dataset_name}"
   project    = "${google_project_service.enable_destination_api.project}"
-
+  location   = "${var.location}}"
```
Comment (aaron-lane) [inline modules/bigquery/main.tf]: @kiran002 there is a typo here which appears to have caused the failure.

```suggestion
  location   = "${var.location}"
```

**Comment by kiran002** [comment]
@aaron-lane Thanks a lot for spotting that! :+1:

**Pull request merged**